content/v2.0/security/https_setup.md

```diff
@@ -0,0 +1,145 @@
+{{% /warn %}}
+
+## Requirements
+
+To enable HTTPS with InfluxDB, you'll need an existing or new InfluxDB instance
+and a Transport Layer Security (TLS) certificate (also known as a Secured Sockets Layer (SSL) certificate).
+
+InfluxDB supports three types of TLS certificates:
+
+* **Single domain certificates signed by a Certificate Authority**
+
+    Single domain certificates provide cryptographic security to HTTPS requests and allow clients to verify the identity of the InfluxDB server.
+    With this certificate option, every InfluxDB instance requires a unique single domain certificate.
+
+* **Wildcard certificates signed by a Certificate Authority**
+
+    Wildcard certificates provide cryptographic security to HTTPS requests and allow clients to verify the identity of the InfluxDB server.
+    Wildcard certificates can be used across multiple InfluxDB instances on different servers.
+
+* **Self-signed certificates**
+
+    Self-signed certificates are _not_ signed by a Certificate Authority (CA).
+    Unlike CA-signed certificates, self-signed certificates only provide cryptographic security to HTTPS requests.
+    They do not allow clients to verify the identity of the InfluxDB server.
+    With this certificate option, every InfluxDB instance requires a unique self-signed certificate.
+    You can generate a self-signed certificate on your own machine.
+
+<!-- InfluxDB supports certificates composed of a private key file (`.key`) and a signed certificate file (`.crt`) file pair, -->
+<!-- as well as certificates that combine the private key file and the signed certificate file into a single bundled file (`.pem`). -->
+
+The following two sections outline how to set up HTTPS with InfluxDB
+[using a CA-signed certificate](#set-up-https-with-a-ca-signed-certificate)
+and [using a self-signed certificate](#set-up-https-with-a-self-signed-certificate).
+
+## Set up HTTPS with a CA-signed certificate
+
+1. **Install the certificate**
+
+    Place the private key file (`.key`) and the signed certificate file (`.crt`) in the `/etc/ssl/` directory.
+
+2. **Set certificate file permissions**
+
+    Users running InfluxDB must have read permissions on the TLS certificate.
+
+    {{% note %}}You may opt to set up multiple users, groups, and permissions.
+    Ultimately, make sure all users running InfluxDB have read permissions for the TLS certificate.
+    {{% /note %}}
+
+    Run the following command to give InfluxDB read and write permissions on the certificate files.
+
+    ```bash
+    sudo chmod 644 /etc/ssl/<CA-certificate-file>
+    sudo chmod 600 /etc/ssl/<private-key-file>
+    ```
+
+3. **Run `influxd` with TLS flags**
+
+    Start InfluxDB with TLS command line flags:
+
+    ```bash
+    influxd \
+    --tls-cert "/etc/ssl/influxdb-selfsigned.crt"\
+    --tls-key "/etc/ssl/influxdb-selfsigned.key"
+    ```
+
+That's it! You've successfully set up HTTPS with InfluxDB.
```

Comment:
Maybe replace this paragraph with a step that confirms https requests are working. They can either try to access the UI using https or try to curl the API.

Reply:
Self-Signed:

`curl -vk https://influxdb:9999/api/v2/ping`

Trusted CA signed:

`curl -v https://influxdb:9999/api/v2/ping`


`-k` is equivalent to `--insecure`